lib/experimental/Information/Headers/index.mdx

```diff
@@ -158,9 +157,17 @@ characteristics:
 2. **Avatar**: Represents entities that can be visualized with an avatar icon.
    (User, company and team)
 3. **Status**: Represent a state or situation associated with the resource, such
-   as "Active," "Inactive," "Pending.”, etc.
+   as "Active," "Inactive," "Pending.", etc.
 4. **List**: Display team members, task owners, or people associated with a
    specific resource.
+5. **Data List**: Display a list of text items that can be collapsed to show
```

Comment:
@desiree-np I added missing metadata types to the documentation. Tell me if you want different wordings here